cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js

```diff
@@ -0,0 +1,89 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TestTypes from '../../../support/dictionary/testTypes';
+import DevTeams from '../../../support/dictionary/devTeams';
+import Permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Users from '../../../support/fragments/users/users';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+
+describe('Data Import › Log details', () => {
+  const testData = {};
+
+  const marcFiles = [
+    {
+      marc: 'oneMarcBib.mrc',
```

Comment:
Use your own files with naming
marcBibC{numberTC}.mrc for  inventory instance
marcAuthC{numberTC}.mrc for marc authority instance